index.d.ts

```diff
@@ -4,7 +4,7 @@
 /**
  * Container for all registered metrics
  */
-export interface register {
+export interface Registry {
```

Comment:
That part should be fine, but I guess this should actually be `export class Registry`. The original `register` constant that's still in index can still be found from line 40, but the type has been changed to this new class.